traefik/Chart.yaml

```diff
@@ -29,3 +29,7 @@ annotations:
     - "feat(deps): update traefik docker tag to v3.1.5"
     - "feat(Traefik Proxy): update rbac following v3.2 migration guide"
     - "chore(release): 🚀 publish v32.1.0"
+#dependencies:
+#  - name: traefik-crds
+#    version: 0.0.1
+#    repository: https://traefik.github.io/charts
```

Comment:
The dependency does not work. I have encountered two issues when testing:
1. schema error on global: 
```less
Error: UPGRADE FAILED: values don't meet the specifications of the schema(s) in the following chart(s):
traefik-crds:
- (root): Additional property global is not allowed
```

2. I allowed additional property in the schema, and another error on values:

```less
helm.go:86: 2024-10-29 16:11:08.371483008 +0100 CET m=+0.316145995 [debug] values don't meet the specifications of the schema(s) in the following chart(s):
traefik:
- traefik-crds.gateway_api: Invalid type. Expected: string, given: boolean
- traefik-crds.hub: Invalid type. Expected: string, given: boolean
```

Reply:
fixed by https://github.com/traefik/traefik-helm-chart/pull/1223/commits/b79e8efbb56815a7e5e22ae8807fc159a9a50323

Reply:
I tested the upgrade and it failed with:

```less
Error: UPGRADE FAILED: Unable to continue with update: CustomResourceDefinition "ingressroutes.traefik.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "traefik"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "traefik"
```

To reproduce:

1. Lauch k3d: `k3d cluster create traefik-hub --port 80:80@loadbalancer --port 443:443@loadbalancer --port 8000:8000@loadbalancer --k3s-arg "--disable=traefik@server:0"`
2. Install Traefik with v32.1.0: `helm install traefik -n traefik --wait   --version v32.1.0   --set ingressClass.enabled=false   --set ingressRoute.dashboard.enabled=true   --set ingressRoute.dashboard.matchRule='Host(`dashboard.docker.localhost`)'   --set ingressRoute.dashboard.entryPoints={web}   --set ports.web.nodePort=30000   --set ports.websecure.nodePort=30001    traefik/traefik`
3. :heavy_check_mark: Check GW API version: `kubectl describe crds httproutes.gateway.networking.k8s.io | head`
4. Uncomment dependency in Chart.yaml with `repository: file://../traefik-crds`
5. Uncomment bottom dependency values (L953-959) about this traefik-crds
6. build dependency: helm dependency build
7. Upgrade and enable GW API provider with local new version: `helm upgrade traefik -n traefik --wait  --set ingressClass.enabled=false   --set ingressRoute.dashboard.enabled=true   --set ingressRoute.dashboard.matchRule='Host(`dashboard.docker.localhost`)'   --set ingressRoute.dashboard.entryPoints={web}   --set ports.web.nodePort=30000   --set ports.websecure.nodePort=30001 --set providers.kubernetesGateway.enabled=true   ./traefik`

Reply:
Please read upgrade documentation: https://github.com/traefik/traefik-helm-chart/pull/1223/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R113